light-client/src/header/eth_header.rs

```diff
@@ -313,7 +311,7 @@ pub fn get_validator_bytes_and_tern_term(extra_data: &[u8]) -> Result<(Validator
         return Err(Error::UnexpectedExtraDataLength(extra_data.len()));
     }
     let num = extra_data[EXTRA_VANITY] as usize;
-    if num == 0 || extra_data.len() <= EXTRA_VANITY + EXTRA_SEAL + num * VALIDATOR_BYTES_LENGTH {
+    if num == 0 || extra_data.len() < EXTRA_VANITY + EXTRA_SEAL + num * VALIDATOR_BYTES_LENGTH {
```

Comment:
Should we care about TURN_LENGTH_SIZE and VALIDATOR_NUM_SIZE?
https://github.com/bnb-chain/bsc/blob/26a4d4fda656cc78436c1931aaea5dc3ed33eeeb/consensus/parlia/parlia.go#L404

Reply:
Since these are assumed to be 1-byte in the current BSC, I do not think it is necessary to consider multi-byte at present.

trun_length
https://github.com/bnb-chain/bsc/blob/26a4d4fda656cc78436c1931aaea5dc3ed33eeeb/consensus/parlia/parlia.go#L58

validator_num
https://github.com/bnb-chain/bsc/blob/26a4d4fda656cc78436c1931aaea5dc3ed33eeeb/consensus/parlia/parlia.go#L399

I think it would be better to deal with this issue after the specification changes such that the number of general validators and trun_length exceeds 256.